src/exam/ExamAPIError.jsx

```diff
@@ -1,66 +1,59 @@
-import React, { useState, useContext } from 'react';
-import { Alert, Icon } from '@edx/paragon';
+import React, { useContext } from 'react';
+import { getConfig } from '@edx/frontend-platform';
+import { Alert, Hyperlink, Icon } from '@edx/paragon';
 import { Info } from '@edx/paragon/icons';
-import { FormattedMessage } from '@edx/frontend-platform/i18n';
+import { injectIntl, intlShape, FormattedMessage } from '@edx/frontend-platform/i18n';
 import ExamStateContext from '../context';
+import messages from './messages';
 
-export default function ExamAPIError() {
+function ExamAPIError({ intl }) {
   const state = useContext(ExamStateContext);
-  const { apiErrorMsg, proctoringSettings } = state;
-  const { contact_us: contactUs, platform_name: platformName } = proctoringSettings;
-  const [showError, setShowError] = useState(false);
+  const { apiErrorMsg } = state;
 
-  const renderHeading = () => {
-    if (contactUs && platformName) {
+  const shouldShowApiErrorMsg = !!apiErrorMsg && !apiErrorMsg.includes('<');
+
+  const renderSupportMsg = () => {
+    const { SITE_NAME, SUPPORT_URL } = getConfig();
+
+    if (SITE_NAME && SUPPORT_URL) {
       return (
         <FormattedMessage
-          id="exam.apiError.text1"
+          id="exam.apiError.supportText.one"
```

Comment:
I am wondering, why we use number on the IDs here? Why not use descriptive text?